yara-x/src/modules/macho/tests/testdata/tiny_macho.out

```diff
@@ -1,21 +1,19 @@
-magic: 0xfeedface
-cputype: 7
-cpusubtype: 3
+magic: 0xcefaedfe
+cputype: 0x7
+cpusubtype: 0x3
 filetype: 2
 ncmds: 2
 sizeofcmds: 136
 flags: 0x1200000
 number_of_segments: 1
+entry_point: 116
 segments:
-  - cmd: 1
-    cmdsize: 56
-    segname: ""
+  - segname: "SP1\300\211\347j\010Wj\001P\260\004\353\260"
```

Comment:
checked to make sure other tools parsed this as such, they do 👍 

I used xmachoviewer to test